org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/correction/ShowAllQuickFixTest.java

```diff
@@ -160,10 +160,10 @@ public void testShowAt() throws Exception {
 			List<Diagnostic> diagnostics = getDiagnostics(cu, astRoot, 4);
 			context.setDiagnostics(diagnostics);
 			codeActions = new CodeActionHandler(this.preferenceManager).getCodeActionCommands(codeActionParams, new NullProgressMonitor());
-			assertEquals(3, codeActions.size());

```

Comment:
Here shows the actions under `CodeActionKind.QuickFix`, we just removed the quick assist of generating getter and setter from fieldDeclaration to `CodeActionKind.Source`, so here the test case should be updated.